core/templates/dev/head/components/forms/schema-based-editors/schema-based-unicode-editor.directive.ts

```diff
@@ -16,6 +16,8 @@
  * @fileoverview Directive for a schema-based editor for unicode strings.
  */
 
+require('dependencies/codemirrorRequires.ts');
```

Comment:
Why is this require here?

Reply:
Ui-codemirror is used in the `schema-based-unicode-editor.directive.html`